homeassistant/components/axis/axis_base.py

```diff
@@ -46,6 +44,12 @@ def __init__(self, event, device):
         super().__init__(device)
         self.event = event
 
+        self._attr_name = f"{device.name} {event.TYPE} {event.id}"
+        self._attr_unique_id = f"{device.unique_id}-{event.topic}-{event.id}"
+
+        self._attr_device_class = event.CLASS
+        self._attr_should_poll = False
```

Comment:
Currently a kinda unwritten standard we are following for static/non-mutables:

```py
class AxisEventBase(AxisEntityBase):
    """Base common to all Axis entities from event stream."""
    
    _attr_should_poll = False
```

Reply:
Great, will fix that tonight

Reply:
Fixed